From 673e78c59aa2d9500e4744127b649e4bd3018f68 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=A4bler?= <oliver.staebler@bytesatwork.ch>
Date: Mon, 13 Jan 2014 13:33:29 +0100
Subject: [PATCH] bytePANEL: Move selection of display

Move the selection of the display type to the submenu of DA8XX (Support
for frame buffer devices).
---
 drivers/video/Kconfig           |   18 ++++++++++++++++++
 drivers/video/backlight/Kconfig |   14 --------------
 2 files changed, 18 insertions(+), 14 deletions(-)

diff --git a/drivers/video/Kconfig b/drivers/video/Kconfig
index cd3a233..4932235 100644
--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -2249,6 +2249,24 @@ config FB_DA8XX_CONSISTENT_DMA_SIZE
 	  The size must be 2MB aligned.
 	  If unsure say 1.
 
+if FB_DA8XX
+choice
+	prompt "bytePANEL Display Type"
+	default LCD_ET070080DH6
+
+config LCD_ET070080DH6
+	bool "Emerging ET070080DH6 (pRGB)"
+	help
+	  If you have an Emerging ET070080DH6 LCD Panel, say Y to enable its driver.
+
+config LCD_DATAIMAGE
+	bool "Data Image 10.4\" (LVDS)"
+	help
+	  If you have an Data Image LCD Panel, say Y to enable its driver.
+
+endchoice
+endif
+
 config FB_VIRTUAL
 	tristate "Virtual Frame Buffer support (ONLY FOR TESTING!)"
 	depends on FB
diff --git a/drivers/video/backlight/Kconfig b/drivers/video/backlight/Kconfig
index ccbb529..df9dac7 100644
--- a/drivers/video/backlight/Kconfig
+++ b/drivers/video/backlight/Kconfig
@@ -125,20 +125,6 @@ config LCD_AMS369FG06
 	  If you have an AMS369FG06 AMOLED Panel, say Y to enable its
 	  LCD control driver.
 
-config LCD_ET070080DH6
-	bool "Emerging ET070080DH6"
-	default y
-	help
-	  If you have an Emerging ET070080DH6 LCD Panel, say Y to enable its
-	  driver.
-
-config LCD_DATAIMAGE
-	bool "Data Image 10.4\""
-	default n
-	help
-	  If you have an Data Image LCD Panel, say Y to enable its driver.
-
-
 endif # LCD_CLASS_DEVICE
 
 #
-- 
1.7.10.4

